packages/api/src/pre-start/env-required.json

```diff
@@ -0,0 +1,19 @@
+[
+  "PORT",
+  "HOST",
+  "ADMINS",
+  "FRONTEND_URL",
+  "MONGO_DB",
+  "MONGO_HOST",
+  "MONGO_PORT",
+  "MONGO_USERNAME",
+  "MONGO_PASSWORD",
+  "COOKIE_DOMAIN",
+  "COOKIE_PATH",
+  "SECURE_COOKIE",
+  "JWT_SECRET",
+  "COOKIE_SECRET",
+  "JWT_ACCESS_EXP",
+  "JWT_REFRESH_EXP",
+  "SERVER_URL"
+]
```

Comment:
- Missing one required env var `NODE_ENV`

Also, curious why you made this file json, rather than just a plain js object? 

Not really that important but I'd say keep it in JS for consistency.

Also, can we add a comment to the top of this file to note that this is where devs should specify required env variables.